egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -90,23 +90,29 @@ def AddPermuteLayer(config_lines, name, input, column_map):
     return {'descriptor': '{0}_permute'.format(name),
             'dimension': input['dimension']}
 
-def AddAffineLayer(config_lines, name, input, output_dim, ng_affine_options = ""):
+def AddAffineLayer(config_lines, name, input, output_dim, ng_affine_options = "", max_change_per_component = 0.25):
```

Comment:
All max-change-per-componnet values have been set to 0.75 (for non-final layers ) / 1.5 (for final layers)